rosco-web/src/test/java/com/netflix/spinnaker/rosco/controllers/V2BakeryControllerTest.java

```diff
@@ -103,20 +105,16 @@ void testSpinnakerHttpExceptionWith404Response() throws Exception {
   }
 
   private static SpinnakerHttpException makeSpinnakerHttpException(int status) {
-    SpinnakerHttpException spinnakerHttpException = mock(SpinnakerHttpException.class);
-    when(spinnakerHttpException.getMessage()).thenReturn("message");
-
-    // Response is a final class, so straightforward mocking fails.
     String url = "https://some-url";
     Response response =
         new Response(
             url,
             status,
             "arbitrary reason",
             List.of(),
-            new TypedString("{ message: \"unused message due to above mock\" }"));
+            new TypedString("{ \"message\": \"unused message due to above mock\" }"));
 
-    when(spinnakerHttpException.getResponse()).thenReturn(response);
-    return spinnakerHttpException;
+    return new SpinnakerHttpException(
+        RetrofitError.httpError(url, response, new JacksonConverter(), String.class));
```

Comment:
How did you choose JacksonConverter?

Reply:
In almost all cases JacksonConverter is used while constructing RestAdaptor in code...

Reply:
Except for [this](https://github.com/square/retrofit/blob/parent-1.9.0/retrofit/src/main/java/retrofit/Platform.java#L70-L72), yes?  Whatever the reason, please include it in a comment in the code.

Reply:
I got it, will replace JacksonConverter with GsonConverter, so no need to add JacksonConverter depedency